package.json

```
@@ -84,6 +84,7 @@
   "devDependencies": {
     "@architect/architect": "^11.1.0",
     "@architect/plugin-lambda-invoker": "^2.0.1",
+    "@architect/sandbox": "github:lpsinger/sandbox#access-key-chars",
```

```suggestion
    "@architect/sandbox": "github:lpsinger/sandbox#nasa-gcn",
```

This branch includes both https://github.com/architect/sandbox/pull/737 and https://github.com/architect/sandbox/pull/738.